tools/README.md

```diff
@@ -0,0 +1,76 @@
+# Envelope tool
+The envelope tool makes it easy to generate firmware envelopes with
+different configurations (like `sdkconfig`).
+
+We call these configurations "variants".
+
+## Toit Host SDK
+The envelopes and the envelope tool need the Toit SDK to be available.
+
+The easiest way to get it is by running
+```
+make download-toit TOIT_VERSION=<some version>
+make build-host
+```
+
+This will checkout the Toit repository into a `toit` directory and
+build the host SDK into `build/host/sdk`.
+
+The `toit.run` and `toit.pkg` are then in `build/host/sdk/bin`.
+
+In the remainder of this document, we assume that you have added
+`build/host/sdk/bin` to your `PATH`. If not, just replace `toit.run`
+and `toit.pkg` with the full path to the binaries.
+
+## The envelope tool
+
```

Comment:
Drop the newlines after # for consistency?

Reply:
Added them for consistency. (See the other comment and https://www.markdownguide.org/basic-syntax/#heading-best-practices)